var/spack/repos/builtin/packages/sst-core/package.py

```diff
@@ -61,7 +62,7 @@ class SstCore(AutotoolsPackage):
     variant("preview", default=False, description="Preview build with deprecated features removed")
     variant("profile", default=False, description="Enable performance profiling of core features")
 
-    depends_on("python", type=("build", "run", "link"))
+    depends_on("python@:3.12", type=("build", "run", "link"))
```

Comment:
```suggestion
    depends_on("python@:3.11", type=("build", "run", "link"))
```
@berquist You comitted this with: "SST does not work with Python 3.12 yet"

If I'm not totally mistaken, you'd have to update this commit to use `python@:3.11` instead.